src/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": "0.1",
```

Comment:
Do we want to use just a version with two elements or a semantic versioning with major, minor, patch?